sdk/liftrqumulo/Client.cs

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Qumulo.Storage
```

Comment:
Is this for dataplane sdk? please remove it from this pr

Reply:
This is for mgmt plane. removed this from PR.